lib/components/Column/Column.treat.ts

```diff
@@ -1,7 +1,24 @@
 import { style } from 'sku/treat';
 
 export const column = style({
-  flexBasis: 0,
-  flexGrow: 1,
   minWidth: 0,
+  overflow: 'hidden',
 });
+
+export const fit = style({ flexShrink: 0 });
+const grow = style({ flexGrow: 1 });
```

Comment:
Feels like a good time to move this to `Box`?

Reply:
> Feels like a good time to move this to `Box`?

It might not work if the parent is not a flexBox? 🤔

Reply:
IMO that's okay. `Box` is conceptually at the same layer as CSS, and CSS has the same issue (if we call it that).